hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -400,43 +395,22 @@ public void deleteBackupInfo(String backupId) throws IOException {
     }
   }
 
-  /*
-   * For postBulkLoadHFile() hook.
-   * @param tabName table name
-   * @param region the region receiving hfile
-   * @param finalPaths family and associated hfiles
+  /**
+   * Registers a bulk load.
+   * @param tabName       table name
+   * @param region        the region receiving hfile
+   * @param cfToHfilePath column family and associated hfiles
    */
-  public void writePathsPostBulkLoad(TableName tabName, byte[] region,
-    Map<byte[], List<Path>> finalPaths) throws IOException {
+  public void registerBulkLoad(TableName tabName, byte[] region,
+    Map<byte[], List<Path>> cfToHfilePath) throws IOException {
     if (LOG.isDebugEnabled()) {
-      LOG.debug("write bulk load descriptor to backup " + tabName + " with " + finalPaths.size()
-        + " entries");
+      LOG.debug("Writing bulk load descriptor to backup {} with {} entries", tabName,
+        cfToHfilePath.size());
     }
     try (Table table = connection.getTable(bulkLoadTableName)) {
-      List<Put> puts = BackupSystemTable.createPutForCommittedBulkload(tabName, region, finalPaths);
+      List<Put> puts = BackupSystemTable.createPutForBulkLoad(tabName, region, cfToHfilePath);
```

Comment:
While we're here, it's probably worth using a BufferedMutator like we did in https://github.com/apache/hbase/pull/6067